indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -96,10 +103,11 @@
                     {% endif %}
                 </div>
                 <div>
-                    {{ render_track(review.track, classes='abstract-track-tag i-tag outline') }}
+                    {{ render_track(review.track) }}
                 </div>
             </div>
-            <div class="i-timeline-item-box header-indicator-top">
+            <div class="i-timeline-item-box header-indicator-top {% if not review.comment %}header-only{% endif %}"
+                 {% if not review.comment %}data-no-comment{% endif %}>
```

Comment:
I think in the last months we always went for not using the camelCase version in `data()` so I'd go for `data('no-comment')` instead. That way it's more consistent and people can easily search for it instead of trying both `no-comment` and `noComment`